iModelCore/iModelPlatform/PublicAPI/DgnPlatform/DgnDb.h

```diff
@@ -63,7 +63,7 @@ enum DgnDbProfileValues : int32_t
     DGNDB_CURRENT_VERSION_Major = 2,
     DGNDB_CURRENT_VERSION_Minor = 0,
     DGNDB_CURRENT_VERSION_Sub1  = 0,
-    DGNDB_CURRENT_VERSION_Sub2  = 4,
+    DGNDB_CURRENT_VERSION_Sub2  = 5,

```

Comment:
why are we updating dgndb profile version?

Reply:
Without updating the DgnDB version there arises an age disparity between the DgnDb and ECDb profile versions.
For example, a bim file with version 2.0.0.4_4.0.0.2_3.1.0.2 will marked as older in age since 4.0.0.2 is no longer the current up-to-date ECDb profile version 
However, since 2.0.0.4 is the current up-to-date DgnDb version, the iModel evolution tests will fail since they'll be checking 2.0.0.4 under the older age test cases where as it is the current up-to-date one.

Reply:
That sounds like a problem with the evolution tests. Afaik nothing in this PR affects dgndb, so there is no need to increment. Unless we say that basing dgndb on a newer ecdb profile warrants incrementing the dgndb profile as well....

Reply:
Yeah, the way the testdb age calculation is set up, seems like that's what it expects. 
We can increment DgnDb version with no consequence, but if we increment ECDb version, the DgnDb needs to be updated as well.

Reply:
You're right we originally defined that an update to the ECDb profile version would require an update to the DgnDb profile version.  I'm not sure there is much value to it but I see no reason to change course now.  We should move the profile version documentation from the internal doc to a markdown file in this repo.  That can be done via an independent PR.